src/transformers/__init__.py

```diff
@@ -51,6 +51,7 @@
 # Base objects, independent of any specific backend
 _import_structure = {
     "benchmark": [],
+    "bitsandbytes_utils": [],
```

Comment:
Let's not make the same mistake as with lots of other utils modules and actually put it in utils/